content/en/docs/tasks/security/automtls/index.md

```diff
@@ -0,0 +1,289 @@
+sleep-58d6644d44-gb55j    2/2     Running   0          38s
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-6f6fc94fb6-8d62h   1/1     Running   0          10s
+httpbin-dcd949489-5fsbs    2/2     Running   0          12s
+NAME                       READY   STATUS    RESTARTS   AGE
+httpbin-54f5bb4957-lzxlg   1/1     Running   0          6s
+sleep-74564b477b-vb6h4     1/1     Running   0          4s
+{{< /text >}}
+
+You should also verify that there is a default mesh authentication policy in the system, which you can do as follows:
+
+{{< text bash >}}
+$ kubectl get policies.authentication.istio.io --all-namespaces
+NAMESPACE      NAME                          AGE
+istio-system   grafana-ports-mtls-disabled   5m
+$ kubectl get meshpolicies.authentication.istio.io
+NAME      AGE
+default   3m
+{{< /text >}}
+
+Last but not least, verify that there are no destination rules that apply on the example services. You can do this by checking the `host:` value of
+ existing destination rules and make sure they do not match. For example:
+
+{{< text bash >}}
+$ kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"
+    host: kubernetes.default.svc.cluster.local
+    host: '*.local'
+    host: grafana.istio-system
+    host: '*.global'
+    host: istio-policy.istio-system.svc.cluster.local
+    host: istio-telemetry.istio-system.svc.cluster.local
+    host: prometheus.istio-system.svc.cluster.local
+    host: prometheus.istio-system
+{{< /text >}}
+
+You can verify setup by sending an HTTP request with `curl` from any `sleep` pod in the namespace `full`, `partial` or `legacy` to either `httpbin.full`, `httpbin.partial` or `httpbin.legacy`. All requests should succeed with HTTP code 200.
+
+For example, here is a command to check `sleep.legacy` to `httpbin.full` reachability:
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n legacy -o jsonpath={.items..metadata.name}) -c sleep -n legacy -- curl http://httpbin.full:8000/ip -s -o /dev/null -w "%{http_code}\n"
+200
+{{< /text >}}
+
+### Start from PERMISSIVE mode
+
+In the setup, we start with `PERMISSIVE` for all services in the mesh.
+
+1. All `httpbin.full` workloads and the workload with sidecar for `httpbin.partial` are able to serve
+both mutual TLS traffic and plain text traffic.
+1. The workload without sidecar for `httpbin.partial` and workloads of `httpbin.legacy` can only serve
+plain text traffic.
+
+Automatic mutual TLS configures the client, `sleep.full`, to send mutual TLS to the first type of
+workloads and plain text to the second type.
+
+You can verify the reachability as:
+
+{{< text bash >}}
+$ for from in "full" "legacy"; do for to in "full" "partial" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl "http://httpbin.${to}:8000/ip" -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
```

Comment:
May be use '/get' or '/headers' that will show the identity in the header if mTLS is used.

Reply:
good point. I didn't know this knob before.

Reply:
done. the oneline is really long line now, but at least we can show mtls from the http output.